utils/chain-ops/genesis/config.go

```diff
@@ -117,6 +119,17 @@ type DeployConfig struct {
 	EIP1559Denominator uint64 `json:"eip1559Denominator"`
 
 	FundDevAccounts bool `json:"fundDevAccounts"`
+
+	// The initial value of the voting delay(unit:block)
+	VotingDelayBlocks uint64 `json:"votingDelayBlocks"`
+	// The initial value of the voting period(unit:block)
+	VotingPeriodBlocks uint64 `json:"votingPeriodBlocks"`
+	// The initial value of the proposal threshold(unit:block)
```

Comment:
The unit of this may not be block, but the number of tokens, right?
```suggestion
	// The initial value of the proposal threshold
```